src/sisl/io/orca/stdout.py

```diff
@@ -15,80 +15,39 @@
 __all__ = ["outputSileORCA", "stdoutSileORCA"]
 
 
+_A = SileORCA.InfoAttr
+
+
 @set_module("sisl.io.orca")
 class stdoutSileORCA(SileORCA):
     """ Output file from ORCA """
 
-    def _setup(self, *args, **kwargs):
-        """ Ensure the class has essential tags """
-        super()._setup(*args, **kwargs)
-        self._completed = None
-        self._na = None
-        self._no = None
-        self._vdw = None
-
-    def readline(self, *args, **kwargs):
-        line = super().readline(*args, **kwargs)
-        if self._completed is None and "ORCA TERMINATED NORMALLY" in line:
-            self._completed = True
-        elif self._completed is None and line == '':
-            self._completed = False
-        elif self._na is None and "Number of atoms" in line:
-            v = line.split()
-            self._na = int(v[-1])
-        elif self._no is None and "Number of basis functions" in line:
-            v = line.split()
-            self._no = int(v[-1])
-        elif self._vdw is None and "DFT DISPERSION CORRECTION" in line:
-            self._vdw = True
-        return line
-
-    readline.__doc__ = SileORCA.readline.__doc__
+    _info_attributes_ = [
+        _A("na", r".*Number of atoms",
+           lambda attr, match: int(match.string.split()[-1])),
+        _A("no", r".*Number of basis functions",
+           lambda attr, match: int(match.string.split()[-1])),
+        _A("_vdw_", r".*DFT DISPERSION CORRECTION",
```

Comment:
Why underscore here instead of just `info.vdw`?

Reply:
Done, changed to `vdw_correction` it was based on the old implementation, but I have now changed to a more meaningful nomenclature. It is not `vdw`, but a `vdw_correction`. Hope you agree.